after/ftplugin/javascriptreact_matchup.vim

```diff
@@ -0,0 +1,25 @@
+" vim match-up - even better matching
+"
+" Maintainer: Andy Massimino
+" Email:      a@normed.space
+"
+
+if !exists('g:loaded_matchup') || !exists('b:did_ftplugin')
+  finish
+endif
+
+let s:save_cpo = &cpo
+set cpo&vim
+
+let b:match_skip = 's:\%(comment\|string\)\%(tsxCloseString\)\@<!'
```

Comment:
should this say tsxCloseString?  is there a jsx equivalent?

Reply:
Oops. Just renamed it in the most recent update. Also retested-- still works as expected